shell/browser/extensions/api/extension_action/extension_action_api.cc

```diff
@@ -11,6 +11,7 @@
 
 #include "base/functional/bind.h"
 #include "base/lazy_instance.h"
```

Comment:
```suggestion
```

Don't see any remaining usages in this file.

Reply:
Good catch. Removed in 987c7ececdabfff9cb605f5429159ff792d2611f